example/udp_client.py

```diff
@@ -7,7 +7,7 @@
 # Start a stream, receiving tensors of shape (640, 480)
 # By default, we listen on port 3333 and send the tensors to the CPU
 # The variable stream can now be `.read()` whenever a tensor is desired
-with UDPInput((640, 480)) as stream:
+with UDPInput((640, 480), "cpu", 4301) as stream:
```

Comment:
potentially instead of string one of the torch "device" functions?

Reply:
You're absolutely right! I've added an overloaded constructor with the correct `torch::Device` type